### PR TITLE
moving main, ucr and synclogs routing via NLB

### PR DESCRIPTION
**Pull request by prem-fissionhq**
https://dimagi-dev.atlassian.net/browse/SAAS-11923

##### ENVIRONMENTS AFFECTED

Staging. 

updated pgbouncer URL to routed Via a new Network Load Balancer. This helps in handling high availability and routing requests to multiple servers.

### CODE REVIEW

File: environments/staging/terraform.yml [hunk 1]
```diff
@@ -219,30 +219,6 @@ servers:
...
-    network_tier: "db-private"
-    az: "a"
-    volume_size: 30
-    volume_encrypted: yes
-    block_device:
-      volume_size: 80
-      encrypted: true
-    group: "couchdb2"
-    os: ubuntu_pro_bionic
-
```
Comment (dannyroberts) [inline environments/staging/terraform.yml:245]: This change (removing these two servers) doesn't seem to fit into this PR. Were these machines already terminated and this is just catching up to that? Either way, would be good to have as a separate PR

File: environments/staging/inventory.ini.j2 [hunk 2]
```diff
@@ -41,11 +41,17 @@ rds_pg0
...
+{{ __pgucr_nlb__ }}
+{{ __pgsynclogs_nlb__ }}
 
 [postgresql:children]
 pgproxy2
 remote_postgresql
 pgformplayer_nlb
+pgmain_nlb
+pgucr_nlb
+pgsynclogs_nlb
```
Comment (dannyroberts) [inline environments/staging/inventory.ini.j2:54]: I think to fix the build you need to add these to `[ansible_skip:children]` as well

File: environments/staging/terraform.yml (hunk 1 above)
Comment (prem-fissionhq) [reply environments/staging/terraform.yml:245]: yes, Danny. These instances are terminated and PR was also merged into master.

**Review by dannyroberts — APPROVED** [review]
